pybossa/syncer/project_syncer.py

```diff
@@ -43,8 +43,10 @@ class ProjectSyncer(Syncer):
     def sync(self, project):
         target = self.get_target(short_name=project.short_name)
         payload = self._build_payload(project, target)
+        is_new_project = False
 
         if not target:
```

Comment:
If the above variable is changed per the comment, you could change this line to the positive case. :)

```python
if is_new_project:
  return self._create(payload, self.target_key)
else:
  ...
```